tests/test_all.py

```diff
@@ -1,43 +1,51 @@
 import subprocess
-import os
-import pytest
 from pathlib import Path
-from saseg_runner import run_egp
-from saseg_runner.runner import SASEGRuntimeError
-SCRIPTDIR = Path(os.path.dirname(__file__)).resolve()
 
-fail_egp = SCRIPTDIR / 'test_fail.egp'
-success_egp = SCRIPTDIR / 'test_success.egp'
-japanese_char_in_path_egp = SCRIPTDIR / '日本語フォルダ名/test_success.egp'
-space_in_path_egp = SCRIPTDIR / 'space exist/test_success.egp'
+import pytest
+import saseg_runner as sas
```

Comment:
I think 'sas' is a little inaccurate since saseg_runner is not sas.

Could you change
・stop using shortened version (e.g write saseg_runner.**** in each line)
or
・import saseg_runner as 'sr' or 'runner' for shortening.

Reply:
Done